### ISSUE TITLE
Issue with falling back from .wav to .ogg in #WAV fields when bmsroot is relative

### ISSUE THREAD

**Issue by PaddyCo** [exch-bms2/beatoraja#481]
In LR2, if an audio file defined in a `#WAV` field is not found, it tries to fall back to a file with the same name but a different extension, for instance:

`#WAV01 sfx.wav` would fall back to sfx.ogg if sfx.wav is not found

beatoraja appears to not do this, so if you re-encode audio as .ogg it will simply stop working.

I created a very simple BMS file to illustrate this [here](https://drive.google.com/file/d/1xKWnoG866L9iUOruj0uR6hzrnV0Zj_E7/view?usp=sharing), this example file only works correctly in LR2

While I feel beatoraja is more correct in this regard, LR2's approach makes it easy for users to reduce file sizes of BMS by converting .wav to .ogg, without having to touch the .bms files (Which I'm assuming would change the sha256 value for the song, making it hard to use with IR)

Sorry if this issue has been raised before 😅

**Comment by Deipnosophy** [comment]
This is strange, your test BMS works for me on Beatoraja 0.7.3 on Windows and I can hear the keysound just fine.

I've converted a lot of BMS to use .ogg files and haven't had any issues, I'm not sure what might be different on your machine (I haven't had any problems on versions before 0.7.3 either).

**Comment by PaddyCo** [comment]
Ah, I tried a clean install and it suddenly worked! 🎉 

After some experimentation it seems the .wav -> .ogg fallback won't work if the songs directory is a relative path (`songs\\test_song`s vs `C:\\songs\\test_songs`), which the configuration tool will set if you add a folder that is inside the beatoraja folder. Maybe there are some known issues regarding relative paths? I have tried rebuilding the BMS database but the issue is still present. 

At least I can now play my entire BMS library without issues as long as I change it to an absolute path👍
